integration/utility_functions.go

```diff
@@ -0,0 +1,524 @@
+package integration
```

Comment:
Just a tiny comment: this file has a massive amount of useful, but disjoint utilities.
I would suggest perhaps splitting them by functionality: you could have a file for form-related operations, one for dkg-related stuff, another for casting votes, etc. 

Ideally, you want to strive for [high cohesion](https://en.wikipedia.org/wiki/Cohesion_(computer_science)) and low coupling at all levels in the software.